content/en/docs/Creating derivatives/Packer/build_azure.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Build Azure images"
+linkTitle: "Build Azure images"
+weight: 4
+date: 2017-01-05
+description: >
+  This section documents the procedure to deploy cOS (or derivatives) images
+  in Azure public cloud provider by using the cOS Vanilla image.
+---
+
+![](https://docs.google.com/drawings/d/e/2PACX-1vSqJWcFThP7K2HS551LCqs73l4ZncXElLjlbCvxY96Ga2Jbjnq79j-DEjaccUZvYEQyphWiDQc9flxk/pub?w=1223&h=691)
+
+Requirements:
+
+* Packer
+* Azure access keys with the appropriate roles and permissions
+* [A Vanilla image](../../../development/build_raw_images#azure)
+* [Packer templates](https://github.com/rancher-sandbox/cOS-toolkit/tree/master/packer)
+
+The suggested approach is based on using Packer templates to customize the
+deployment and automate the upload and publish to Azure of cOS derivatives or cOS itself. For all the details
+and possibilities of Packer check the [official documentation](https://www.packer.io/guides/hcl).
+
+## Run the build with Packer
+
+Publishing an image in Azure based on top of the latest cOS Vanilla image is
+fairly simple. In fact, it is only needed to set the Azure credentials
+and run a `packer build` process to trigger the deployment and register the
+resulting snapshot as an image. In such case the latest cOS image will be
```

Comment:
we should maybe underline that it is required to upload the image before this. And maybe link to the docs on how to upload it

Reply:
it is already in the requirements section with a link to the appropriate doc.

Reply:
my bad, I mean a link to https://rancher-sandbox.github.io/cos-toolkit-docs/docs/getting-started/booting/#importing-an-azure-image-manually